services/graph/pkg/service/v0/users.go

```diff
@@ -171,6 +173,35 @@ func (g Graph) GetUser(w http.ResponseWriter, r *http.Request) {
 			errorcode.GeneralException.Render(w, r, http.StatusInternalServerError, err.Error())
 		}
 	}
+	sel := strings.Split(r.URL.Query().Get("$select"), ",")
+	exp := strings.Split(r.URL.Query().Get("$expand"), ",")
+	if slices.Contains(sel, "drive") || slices.Contains(sel, "drives") || slices.Contains(exp, "drive") || slices.Contains(exp, "drives") {
+		wdu, err := url.Parse(g.config.Spaces.WebDavBase + g.config.Spaces.WebDavPath)
+		// TODO: filter this request for the user
+		f := listStorageSpacesUserFilter(user.GetId())
+		lspr, err := g.gatewayClient.ListStorageSpaces(r.Context(), &storageprovider.ListStorageSpacesRequest{
+			Opaque:  nil,
+			Filters: []*storageprovider.ListStorageSpacesRequest_Filter{f},
+		})
+		if err != nil {
+			g.logger.Err(err).Interface("query", r.URL.Query()).Msg("error getting storages")
+		}
```

Comment:
we always need to check not only protocol errors but also Status code on the response.

Reply:
We can always return an user without a drives object if the return code from grpc is `!Code_CODE_OK`